experiments/ClimaEarth/test/runtests.jl

```diff
@@ -2,8 +2,6 @@ using SafeTestsets
 import ClimaComms
 ClimaComms.@import_required_backends
 
-gpu_broken = ClimaComms.device() isa ClimaComms.CUDADevice
```

Comment:
Nice!

Reply:
We can remove it from `test/runtests.jl` too

Reply:
Add it in one of my other open PRS